src/xpcc/architecture/platform/driver/core/avr/ram.cpp.in

```diff
@@ -76,9 +77,9 @@ initializeMemory(void)
 void *
 xpcc::avr::allocateMemory(std::size_t requestedSize)
 {
-	if (requestedSize == 0 ||
-		requestedSize > MAX_BLOCK_PARTS * BLOCK_SIZE)
+	if (requestedSize == 0 || requestedSize > MAX_BLOCK_PARTS * BLOCK_SIZE)
 	{
+		xpcc_assert(0, "core", "heap", "malloc", requestedSize);
```

Comment:
that is against the standard though, right?
shouldn't malloc as defined in the C and C++ standards return a `0` or a `nullptr` in case it runs out of memory so that the user code can then decide what to do?